### PR TITLE
Update Ubuntu series

### PR DESCRIPTION
**Pull request by AlanGriffiths**
We no longer need 18.04 and 20.10 has been released

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ env:
   jobs:
   - TASK=google:ubuntu-20.04-64:spread/build/sbuild:debian_sid
   - TASK=google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_devel
-  - TASK=google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_bionic
+  - TASK=google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_groovy
   - TASK=google:ubuntu-20.04-64:spread/build/sbuild:ubuntu
   - TASK=google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_proposed
   - TASK=google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_arm64
```
Comment (Saviq) [inline .travis.yml:51]: Not sure what's the order I put these in…

File: .travis.yml [hunk 2]
```diff
@@ -69,7 +69,7 @@ jobs:
   include:
   - &ppa-upload
     stage: upload to ppa:mir-team
-    env: RELEASE=18.04
+    env: RELEASE=20.04
```
Comment (Saviq) [inline .travis.yml:72]: You want to replace `20.04` with `20.10` below?

File: .travis.yml [hunk 3]
```diff
@@ -119,6 +119,7 @@ jobs:
     script:
     - tools/process_snaps.py
   allow_failures:
-    - env: TASK=google:ubuntu-devel-proposed-64:spread/build/ubuntu:amd64
+    - env: TASK=google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_devel
+    - env: TASK=google:ubuntu-20.04-64:spread/build/sbuild:ubuntu_proposed
```
Comment (Saviq) [inline .travis.yml:123]: Thanks.

File: spread.yaml [hunk 4]
```diff
@@ -25,11 +25,11 @@ environment:
...
     RELEASE: focal
     RELEASE/ubuntu: focal
-    RELEASE/ubuntu_devel: groovy
-    RELEASE/ubuntu_bionic: bionic
+    RELEASE/ubuntu_devel: hirsute
+    RELEASE/ubuntu_groovy: groovy
     RELEASE/debian_sid: sid
     MIR_PPA: false
-    MIR_PPA/ubuntu_bionic: true
+    MIR_PPA/ubuntu_focal: true
```
Comment (Saviq) [inline spread.yaml:32]: This should actually go away altogether, along with the adding of the PPA in `spread/…`.

Note `/ubuntu_focal` here wouldn't match anything.

**Review by Saviq — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (AlanGriffiths) [reply .travis.yml:72]: Done

File: spread.yaml (hunk 4 above)
Comment (AlanGriffiths) [reply spread.yaml]: Done

**Review by Saviq — APPROVED** [review]
Yup!

bors merge

**Pull request merged**